src/getLookupResults.js

```diff
@@ -71,6 +84,76 @@ const getLookupResults = async (entities, options, requestWithDefaults, Logger)
   return lookupResults.concat(ignoredIpLookupResults);
 };
 
+function _isEntityBlocklisted(entity, options, Logger) {
+  const blocklist = options.blocklist;
```

Comment:
Change to mimic the changes made in the other MISP PR [here](https://github.com/polarityio/misp/commit/48e1822223c711275c4c501739aa4d40334c368e#diff-0d458569bd1c86b86a54927f40fa1fc49fa9862cc6b0a10adbfcd780e9286327R518-R556).  Functions that directly return boolean values (e.i. true false) and unnamed complex conditionals are a code smells